blueocean-dashboard/src/main/js/components/InputParameters.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component, PropTypes } from 'react';
+import isoFetch from 'isomorphic-fetch';
+import {
+  i18nTranslator,
+  ToastUtils,
+} from '@jenkins-cd/blueocean-core-js';
+import {
+  ModalView,
+  ModalBody,
+  ModalHeader,
+} from '@jenkins-cd/design-language';
+import { supportedInputTypesMapping } from './parameter/index';
+
+/**
+ * Simple helper to stop stopPropagation
+ * @param event the event we want to cancel
+ */
+const stopProp = (event) => {
+    event.stopPropagation();
+};
+
+/**
+ * Translate function
+ */
+const t = i18nTranslator('blueocean-dashboard');
+
+/**
+ * Creating a "<form/>"less form to submit the input parameters requested by the user in pipeline.
+ *
+ * We keep all form data in state and change them onChange and onToggle (depending of the parameter
+ * type). We match the different supported inputTypes with a mapping functions
+ * @see supportedInputTypesMapping
+ * That mapping delegates to the specific implementation where we further delegate to JDL components.
+ * In case you want to register a new mapping you need to edit './parameter/index' to add a new mapping
+ * and further in './parameter/commonProptypes' you need to include the new type in the oneOf array.
+ */
+export default class InputParameters extends Component {
```

Comment:
tiny thing, but I worry that the name "Input" may confuse people looking at code for input vs params (they are very similar). It says parameters so this is ok... perhaps "ParameterForm" or something? just a suggestion. Nice commentary ;)